tls/tls_srv.c

```diff
@@ -74,7 +74,7 @@ ttls_parse_servername_ext(TlsCtx *tls, const unsigned char *buf, size_t len)
 
 	servername_list_size = ((buf[0] << 8) | (buf[1]));
 	if (unlikely(servername_list_size + 2 != len)) {
-		T_DBG("ClientHello: bad SNI list size\n");
+		T_WARN("ClientHello: bad SNI list size\n");
```

Comment:
Why do we need the patch " TLS server: raise handshake errors log severity "? The first claim in #1511 is about 2 meaningless lines in the log: we need not only to make the log messages more meaningful, but also reduce the number of records in the log. The kerne log (dmesg) isn't cheap and even now, experiencing only moderate security scans, we have plenty of log messages. If we increase the number of error messages, then the system becomes hardly monitored under more or less intensive malicious traffic raising many TLS error messages.

Reply:
We cannot print both details and peer IP address on a single line. That is the reason for 2 lines: one prints details and another prints IP address. SNI code is implemented inside FW layer and avoids this problem. But in TLS layer it's not valid to use something like that:
```
TfwCliConn *cli_conn = &container_of(ctx, TfwTlsConn, tls)->cli_conn;
```

Reply:
Just noticed "We can get the client IP from the TlsCtx->io_in.skb_list.". Will try that.

Reply:
We're going to propose Tempesta TLS for the Linux upstream, so it must be slef-sufficient and do not require any modules like Tempesta FW, even for logging. That might be not so trivial to get IP address from the skb list, but the good point is that we pretty sure that at the TLS layer TCP header are already processed (i.e. the kernel already has called `pskb_copy()` and the TCP header is in a contiguous memory).

Reply:
Ok, maybe consider refactoring TLS logging as `TlsCtx-> error_log()` indirect call then? Or buffer some limited amount of messages for an upper layers? With # 537 in mind we shouldn't hardcode logging back-end into TLS anyway.